packages/react-dom/src/__tests__/ReactDOMTextarea-test.js

```diff
@@ -359,10 +359,10 @@ describe('ReactDOMTextarea', () => {
             {'there'}
           </textarea>,
         ),
-      ).toWarnDev(
-        'Use the `defaultValue` or `value` props instead of setting children on <textarea>.',
-      );
-    }).toThrow();
+      ).toThrow();
```

Comment:
While we’re at it, can we add a message here so that we don’t pass on any error?